crates/parcel/src/request_tracker/request_tracker.rs

```diff
@@ -255,7 +260,7 @@ impl RequestTracker {
     match request_node {
       RequestNode::Root => Err(diagnostic_error!("Impossible")),
       RequestNode::Incomplete => Err(diagnostic_error!("Impossible")),
-      RequestNode::Error(error) => Err(anyhow!(error.clone())),
+      RequestNode::Error(error) => Err(anyhow!(error.to_string())),
```

Comment:
I believe this will butcher output, specifically stack traces will be missing when enabled as an env var

Reply:
It's the other way around actually ; before this diff `RequestNode::Error` is a string, 

Now I'm carrying the inner error but we still need to convert to string here unfortunately. We needed a clonable error type to fix it.

I was changing because atm strack-traces are missing.

Reply:
Oh right, I thought that was fixed. We should revisit this at some point.